src/astro/databases/snowflake.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+from airflow.providers.snowflake.hooks.snowflake import SnowflakeHook
+from pandas.io.sql import SQLDatabase
+
+from astro.constants import DEFAULT_CHUNK_SIZE, LoadExistStrategy
+from astro.databases.base import BaseDatabase
+from astro.sql.tables import Table
+from astro.utils.dependencies import pandas_tools
+
+DEFAULT_CONN_ID = SnowflakeHook.default_conn_name
+
+
+class SnowflakeDatabase(BaseDatabase):
+    """
+    Handle interactions with Postgres databases. If this class is successful, we should not have any Postgres-specific
```

Comment:
Docstrings needs to be updated to Snowflake :) -- currently it talks about Postgres

Reply:
Opps, done. :)